physics/cires_ugwp_triggers_v1.F90

```diff
@@ -0,0 +1,562 @@
+        subroutine ugwp_triggers
+        implicit none
+        write(6,*) ' physics-based triggers for UGWP ' 
+        end subroutine ugwp_triggers
+!	
+     SUBROUTINE  subs_diag_geo(nx, ny,  lat, lon, rlat, rlon, dy, dx,  &
+                               cosv, rlatc, brcos, brcos2, dlam1, dlam2, dlat, divJp, divJm)
+      use ugwp_common , only : deg_to_rad
+      
+      implicit none
+       integer :: nx, ny
+       real    :: lon(nx), lat(ny)
+       real    :: rlon(nx), rlat(ny) , cosv(ny), tanlat(ny)  
+       real    :: rlatc(ny-1), brcos(ny), brcos2(ny)              
+       real    :: earth_r, ra1, ra2, dx, dy, dlat
+       real    :: dlam1(ny), dlam2(ny),  divJp(ny), divJm(ny)
+       integer :: j
+!
+!    specify common constants and
+!    geometric factors to compute deriv-es etc ...
+!    coriolis coslat tan etc...
+!
+      earth_r = 6370.e3
+      ra1     = 1.0 / earth_r
+      ra2     = ra1*ra1
+!
+      rlat   = lat*deg_to_rad
```

Comment:
some of these constants  exist in FV3. CCPP can handle conversion from radian to degree - if you only need them in degree and never in radian, you can get them in degree from the host model instead of in radian